### PR TITLE
Feature/sorting

### PR DESCRIPTION
**Pull request by jarmovdh**

### CODE REVIEW

File: src/components/time-entries/TimeEntries.tsx [hunk 1]
```diff
@@ -17,41 +19,70 @@ export const TimeEntries = ({ isModalActive, handleModal }: Types.AtBuildProps)
...
-              weekday: "long",
-              day: "numeric",
-              month: "numeric",
-            });
+        <Styled.Container>
+          <Styled.Label>Sort by Date:</Styled.Label>
+          <Styled.Select value={sortOrder} onChange={(e) => setSortOrder(e.target.value)}>
+            <option value="">--Select option--</option>
+            {sortOptions.map((label) => (
+              <option key={Math.random()} label={label} value={label}>
```
Comment (markbolier) [inline src/components/time-entries/TimeEntries.tsx:50]: Maybe change the `Math.random()` on the key to a more robust solution

**Review by markbolier — COMMENTED** [review]
Looks good! ✨

File: src/components/team-member-entries/TeamMemberEntries.tsx [hunk 2]
```diff
@@ -32,9 +53,18 @@ export const TeamMemberEntries = ({ isModalActive, handleModal }: Types.AtBuildT
...
           ))}
         </Styled.Select>
+        <Styled.Label>Sort by name:</Styled.Label>
+        <Styled.Select value={sortOrder} onChange={(e) => setSortOrder(e.target.value)}>
+          <option value="">--Select option--</option>
+          {sortOptions.map((label) => (
+            <option key={Math.random()} label={label} value={label}>
+              {label}
+            </option>
+          ))}
```
Comment (amijsb) [inline src/components/team-member-entries/TeamMemberEntries.tsx:63]: Replace `Math.random()` with `label + i`: 
```suggestion
          {sortOptions.map((label, i) => (
            <option key={label + i} label={label} value={label}>
              {label}
            </option>
          ))}
```

File: src/components/time-entries/TimeEntries.tsx [hunk 3]
```diff
@@ -17,41 +19,70 @@ export const TimeEntries = ({ isModalActive, handleModal }: Types.AtBuildProps)
...
+                {label}
+              </option>
+            ))}
```
Comment (amijsb) [inline src/components/time-entries/TimeEntries.tsx:53]: See my previous comment.

**Review by amijsb — CHANGES_REQUESTED** [review]
Nice! Good busy 🚀

File: src/components/time-entries/TimeEntries.tsx [hunk 4]
```diff
@@ -17,41 +19,70 @@ export const TimeEntries = ({ isModalActive, handleModal }: Types.AtBuildProps)
...
-          .map((timeEntry, i, arr) => {
-            const currentDate = new Date(timeEntry.startTime).toLocaleDateString("en-EN", {
-              weekday: "long",
-              day: "numeric",
-              month: "numeric",
-            });
+        <Styled.Container>
+          <Styled.Label>Sort by Date:</Styled.Label>
+          <Styled.Select value={sortOrder} onChange={(e) => setSortOrder(e.target.value)}>
+            <option value="">--Select option--</option>
```
Comment (sabinewst) [inline src/components/time-entries/TimeEntries.tsx:48]: Add `disabled` here, value should be either ascending or descending and cannot be empty.

**Review by sabinewst — CHANGES_REQUESTED** [review]

**Review by sabinewst — APPROVED** [review]

**Review by amijsb — APPROVED** [review]
🥇

**Pull request merged**